src/test/resources/features/RFMCYAPage.feature

```diff
@@ -60,62 +61,83 @@ Feature: RFM check your answers page
     Then I should be on RFM CYA Page
+    Then I should see Report Pillar 2 top-up taxes text is not clickable
     When I click on Save&Continue button
     Then I should be on RFM Saving Progress Page
+    Then I should see Report Pillar 2 top-up taxes text is not clickable
     When I click on Continue button
     And I access RFM corporate position page
+    Then I should see Report Pillar 2 top-up taxes text is not clickable
     When I select corp position as NFM
     And I click on Continue button
     Then I should be on New NFM guidance page
+    Then I should see Report Pillar 2 top-up taxes text is not clickable
     When I click on Continue button
     Then I should be on RFM registered in UK page
+    Then I should see Report Pillar 2 top-up taxes text is not clickable
     And I select option Yes and continue to next
     Then I should be on RFM UK based entity type page
+    Then I should see Report Pillar 2 top-up taxes text is not clickable
     When I select option UK limited company and continue to GRS page
     And I click on Continue button
     And I click on Save&Continue button
     Then I should be on RFM Contact Guidance page
+    Then I should see Report Pillar 2 top-up taxes text is not clickable
     And I click on Continue button
     Then I should navigate to RFM Contact Detail Page
+    Then I should see Report Pillar 2 top-up taxes text is not clickable
     When I provide RFM contact name as RFM test contact
     And I click on Continue button
     Then I should navigate to RFM Contact Email Page
+    Then I should see Report Pillar 2 top-up taxes text is not clickable
     When I provide RFM contact email as rfm@email.com
     And I click on Continue button
     Then I should navigate to RFM Contact Number Page
+    Then I should see Report Pillar 2 top-up taxes text is not clickable
     And I select option Yes and continue to next
     Then I should navigate to RFM Contact Input Page
+    Then I should see Report Pillar 2 top-up taxes text is not clickable
     When I provide RFM contact number as 01632960001
     And I click on Continue button
     Then I should navigate to RFM Second Contact Question Page
+    Then I should see Report Pillar 2 top-up taxes text is not clickable
     And I select option No and continue to next
     Then I should be on RFM Contact Address Page
+    Then I should see Report Pillar 2 top-up taxes text is not clickable
     And I enter Address Line 1 as Address Line 1 CYA
     And I enter City as City CYA
     And I enter Postal Code as EH5 5WY
     And I enter Country as Australia
     And I click on Country selected
     And I click on Continue button
     Then I should navigate to RFM Final Review Page
+    Then I should see Report Pillar 2 top-up taxes text is not clickable
     When  I click change link for RFM Change Second Contact Preference
     And I select option Yes and continue to next
+    Then I should see Report Pillar 2 top-up taxes text is not clickable
```

Comment:
Do we really need to validate Service banner on each and every page of RFM?

Reply:
this is to make sure its not clickable on any of the RFM screens